content/upcoming/enry.md

```diff
@@ -0,0 +1,181 @@
+---
+author: mcarmonaa
+date: 2017-07-03
+draft: true
+title: "enry: detecting your accent"
+image: ""
+description: "Announcing enry, a faster implementation of github/linguist in Go for programming language detection"
+categories: ["technical"]
+---
+
+## enry (⊙.☉)?
+If you don't know yet what *enry* is looking at the so descriptive name of "enry", maybe it's because you aren't the kind of person keen on films. Don't worry about it and keep reading.
+
+*[enry](https://github.com/src-d/enry)* is a tool written in Go to perform file language detection. It started as a port of *[github's project linguist](https://github.com/github/linguist)*.
+
+With all the tools that *enry*  provides,  you can know what language a file was written in, whether this file is used as documentation for a project, is a vendor file or even if it is a binary file.
```

Comment:
I think *with the tools that enry provides* (without the "all") or "enry provides you with tools to" sounds better. Also, double space after the comma.